tests/tests/tier0/bluechi-change-port-with-p-cmd-option/test_bluechi_change_port_with_p_cmd_option.py

```diff
@@ -0,0 +1,70 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+from typing import Dict
+import os
+
+from bluechi_test.test import BluechiTest
+from bluechi_test.machine import BluechiControllerMachine, BluechiAgentMachine
+from bluechi_test.config import BluechiControllerConfig, BluechiAgentConfig
+from bluechi_test.systemctl import SystemCtl
+
+NODE_FOO = "node-foo"
+
+
+def exec(ctrl: BluechiControllerMachine, nodes: Dict[str, BluechiAgentMachine]):
+    node_foo = nodes[NODE_FOO]
+    bluechi_agent_str = "bluechi-agent"
+    bluechi_controller_str = "bluechi-controller"
+
+    # change controller port to 8421
+    ctrl.exec_run(
+        f"sed -i 's|ExecStart=/usr/libexec/{bluechi_controller_str}|ExecStart=/usr/libexec/{bluechi_controller_str} -p 8421|' /usr/lib/systemd/system/{bluechi_controller_str}.service"
+    )
+    ctrl_sysctl = SystemCtl(ctrl)
+    ctrl_sysctl.daemon_reload()
+    ctrl_sysctl.restart_unit(bluechi_controller_str)
+    assert ctrl.wait_for_unit_state_to_be(bluechi_controller_str, "active")
+
+    # Check if port 8421 is listenning
+    _, output = ctrl.exec_run("lsof -i:8421")
+    assert "bluechi-c" in str(output)
```

Comment:
This assertion failed on testing farm. systemd considers the bluechi-controller active as soon as it is forked off (see [here](https://www.freedesktop.org/software/systemd/man/latest/systemd.service.html#Type=) for type simple). So it can happen that bluechi-controller.service is active, but hasn't started listening yet. Adding a short `time.sleep(0.5)` before should suffice.

Reply:
done, thx

Reply:
@engelmi I tried to add 10 seconds to the waiting time, still fail in the same way，
when i try to reproduce the error, i noticed in testing-farm the image is`fedora-rawhide`. so in my local env i use `centos-stream` image, there are no mistakes, then i changed the image to `fedora-rawhide` in my local env, errors also occurred as a result.
So, could this be the cause of the problem?

Reply:
I also have the valgrind test failing in #838 
Something weird, on it, will let you know when I figure it out

Reply:
Okay, somehow when doing `bluechictl restart` - when run with valgrind something's off with state change/monitor messages. I will try to dig a bit deeper, but for this specific case you can replace `restart` with `stop` and `start` - see that it works to pinpoint the issue

Reply:
thanks for the help @mkemel but i replace restart with stop and start, still got the same error 😮‍💨

Reply:
opps, when i run locally with `-eWITH_VALGRIND=1`, it failed in same manner as in testing-farm. the output of `lsof` actually is:
`memcheck- ...  TCP *:8421 (LISTEN)` not `bluechi-c`

Reply:
No, it's not. Interesting

Reply:
in this commit, yes..because the wrong `sed` commands, nothing changed